### PR TITLE
(litmus-portal): Added header and related components.

### LINKED ISSUE #1571: Header, Notifications and Profile Section Dev.
Header, Notifications and Profile Section development for Litmus Portal.

### PR DESCRIPTION
**Pull request by ishangupta-ds**
Signed-off-by: ishangupta-ds <ishan.gupta@mayadata.io>

What this PR does / why we need it:

Updated SideBar and BreadCrumbs. Added profile icon, header, profile drop-down and notifications drop-down.

Which issue this PR fixes:

fixes #1571 

**Checklist**

* [x] Does this PR have a corresponding GitHub issue?
* [ ] Have you included relevant README for the chaoslib/experiment with details?
* [ ] Have you added debug messages where necessary? 
* [ ] Have you added task comments where necessary? 
* [ ] Have you tested the changes for possible failure conditions?
* [ ] Have you provided the positive & negative test logs for the litmusbook execution?
* [ ] Does the litmusbook ensure idempotency of cluster state?, i.e., is cluster restored to original state?
* [ ] Have you used non-shell/command modules for Kubernetes tasks?
* [ ] Have you (jinja) templatized custom scripts that is run by the litmusbook, if any? 
* [ ] Have you (jinja) templatized Kubernetes deployment manifests used by the litmusbook, if any?
* [ ] Have you reused/created util functions instead of repeating tasks in the litmusbook?
* [ ] Do the artifacts follow the appropriate directory structure? 
* [ ] Have you isolated storage (eg: OpenEBS) specific implementations, checks? 
* [ ] Have you isolated platform (eg: baremetal kubeadm/openshift/aws/gcloud) specific implementations, checks?  
* [ ] Are the ansible facts well defined? Is the scope explicitly set for playbook & included utils?
* [ ] Have you ensured minimum/careful usage of shell utilities (awk, grep, sed, cut, xargs etc.,)?
* [ ] Can the limtusbook be executed both from within & outside a container (configurable paths, no hardcode)?
* [ ] Can you suggest the minimal resource requirements for the litmusbook execution?
* [ ] Does the litmusbook job artifact carry comments/default options/range for the ENV tunables?
* [ ] Has the litmusbooks been linted? 

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by ishangupta-ds** [comment]
Review at the earliest i need to update community page and time series after this merge. @rajdas98 @arkajyotiMukherjee @gdsoumya

File: litmus-portal/frontend/src/components/CustomBreadCrumbs/index.tsx [hunk 1]
```diff
@@ -12,19 +13,22 @@ const CustomBreadcrumbs: React.FC<CustomBreadcrumbsProps> = ({ location }) => {
...
-    <Breadcrumbs
-      separator={<NavigateNextIcon fontSize="small" />}
-      aria-label="breadcrumb"
-    >
-      <Link to="/" className={classes.breadCrumb}>
-        Home
-      </Link>
+    <Breadcrumbs aria-label="breadcrumb">
+      <Link to="/" className={classes.breadCrumb} />
       {path.map((p: string) => {
```
Comment (rajdas98) [inline litmus-portal/frontend/src/components/CustomBreadCrumbs/index.tsx:21]: it should be `paths.map((path: string))`

File: litmus-portal/frontend/src/components/Header/index.tsx [hunk 2]
```diff
@@ -1,20 +1,149 @@
...
+          'https://res.cloudinary.com/practicaldev/image/fetch/s--jZgtY8cn--/c_imagga_scale,f_auto,fl_progressive,h_1080,q_auto,w_1080/https://res.cloudinary.com/practicaldev/image/fetch/s--x3KZoo7u--/c_imagga_scale%2Cf_auto%2Cfl_progressive%2Ch_420%2Cq_auto%2Cw_1000/https://dev-to-uploads.s3.amazonaws.com/i/0v6zstfufm96e09isqo6.png',
+      },
+    ];
+
+    const iterations = notificationsList.length;
+
+    localStorage.setItem('#ActiveMessages', `${notificationsList.length}`);
+
+    const oneDaySeconds = 60 * 60 * 24;
+    let curUnix = Math.round(
```
Comment (rajdas98) [inline litmus-portal/frontend/src/components/Header/index.tsx:74]: Add line before and after of curUnix function

File: litmus-portal/frontend/src/components/NotificationDropdown/index.tsx [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+  const handleClick = useCallback(() => {
+    setIsOpen(!isOpen);
+  }, [isOpen, setIsOpen]);
+
+  const handleClickAway = useCallback(() => {
+    setIsOpen(false);
+  }, [setIsOpen]);
+
+  const id = isOpen ? 'scroll-playground' : null;
+  return (
```
Comment (rajdas98) [inline litmus-portal/frontend/src/components/NotificationDropdown/index.tsx:42]: Add line before the return function

File: litmus-portal/frontend/src/components/CustomBreadCrumbs/index.tsx (hunk 1 above)
Comment (rajdas98) [reply litmus-portal/frontend/src/components/CustomBreadCrumbs/index.tsx:21]: don't put single character variable

File: litmus-portal/frontend/src/components/NotificationDropdown/index.tsx (hunk 3 above)
Comment (ishangupta-ds) [reply litmus-portal/frontend/src/components/NotificationDropdown/index.tsx:44]: Done.

File: litmus-portal/frontend/src/components/Header/index.tsx (hunk 2 above)
Comment (ishangupta-ds) [reply litmus-portal/frontend/src/components/Header/index.tsx:75]: Done.

File: litmus-portal/frontend/src/components/CustomBreadCrumbs/index.tsx (hunk 1 above)
Comment (ishangupta-ds) [reply litmus-portal/frontend/src/components/CustomBreadCrumbs/index.tsx]: Changed.

**Comment by ishangupta-ds** [comment]
> Add a line before and after of each function and HTML tag

Added.

**Comment by ishangupta-ds** [comment]
@rajdas98 verify the changes.

**Pull request merged**